go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/cenkalti/backoff v2.0.0+incompatible
 	github.com/cpuguy83/go-md2man v1.0.8 // indirect
 	github.com/dustin/go-humanize v0.0.0-20180713052910-9f541cc9db5d
-	github.com/exoscale/egoscale v0.14.3
+	github.com/exoscale/egoscale v0.14.4-0.20190401095233-3ad41b3e71e6
```

Comment:
You can use `github.com/exoscale/egoscale v0.15.0` now

Reply:
done ;)